plugins/techdocs/src/home/components/Tables/EntityListDocsTable.tsx

```diff
@@ -38,6 +38,7 @@ import { DocsTableRow } from './types';
  * @public
  */
 export type EntityListDocsTableProps = {
+  options?: object;
```

Comment:
Could you create a specific TableOptions type to help people understand what properties can be set on this object?

Reply:
thanks for the response, updated now to reflect `Options` from `@material-table/core`